### PR TITLE
Nehar example

### PR DESCRIPTION
**Pull request by ben9809**

### CODE REVIEW

**Comment by gurgese** [comment]
@jasonlyik The NeHAR example is ready could you review this PR?

**Comment by jasonlyik** [comment]
Hey @ben9809, apologies that it took a while but I got around to reviewing the NeHAR example. All looks good except that I can't run the SCNN from my M1 Mac since there is some pytorch issue. 

Remaining are very small changes for the ipynb which I've commented.

**Comment by ben9809** [comment]
Hello @jasonlyik, are you encountering issues specifically related to the training script on your Mac?

**Comment by jasonlyik** [comment]
@ben9809 The error appears to be a conv issue on M1 Mac, anytime the forward pass is called.
I can reproduce the same error with the simple script:

```
conv = nn.Conv1d(6, 200, 2, padding=4)
x = torch.rand(256, 6, 1)
y = conv(x)
```
So training or inference of the CSNN architecture is not possible. I could train and run the other two models in the notebook though.

**Comment by jasonlyik** [comment]
@ben9809 I didn't associate them with any line numbers, copy pasting them here:

- Error with first block that contains `settings` dict, it doesn't contain keys "alpha_hid" and "alpha_out" which are used in the snn.Synaptic layers just afterwards
- Also small change needed to update NeuroBench metric from "multiply_accumulates" to "synaptic_operations", as well as formatting the pd.DataFrame

**Comment by ben9809** [comment]
Thank you @jasonlyik!

In order to maintain compatibility with the version from PyPI (used in Google Colab) and the version in the dev branch, I've decided to omit "synaptic_operations" as a metric extracted from the different models in the notebook for now.

Let me know what you think about this decision.

**Comment by jasonlyik** [comment]
Sounds good, we have some other changes planned though for the v1.0 release so I may go through and touch up later as well.

I'll verify and pull this in, thanks!

**Review by jasonlyik — APPROVED** [review]
